source/pkcore/faq.rst

```diff
@@ -9,6 +9,12 @@ What Spigot version does the current plugin version support?
 
 ProjectKorra Core **v1.8.8** is compatible with Spigot **1.13.2** at the moment.
 
+If it still is not operating properly, consider that your Spigot version may not be fully up to date. Spigot **1.13.2** alone has over **400** updates to it. You can check the status of your version by executing the ``/version`` command while on your server. If you are not up to date, try getting an updated Spigot ``.jar`` file. You can achieve this by building the latest version with Build Tools.
```

Comment:
This is supposed to give people a quick short answer, and while I agree there should be some information on how to get an updated Spigot jar, this is very verbose and needs to be cut down.

- Change ` consider that your Spigot version may not be fully up to date.` to `it may be due to incompatible versions. Please ensure that you are using Spigot or Paper **1.13.2**.`
- Remove `Spigot **1.13.2** alone has over **400** updates to it.`
- Change `You can check the status of your version by executing the ``/version`` command while on your server.` to `You can check your jar's version by executing the ``/version`` command.
-  Change `If you are not up to date, try getting an updated Spigot ``.jar`` file. You can achieve this by building the latest version with Build Tools.` to `If it isn't a recent **1.13.2** jar file, please use the latest version of BuildTools to build one.`